packages/web/components/PageHeader.tsx

```diff
@@ -217,9 +243,30 @@ export const PageHeader: React.FC = () => {
             right="0"
             onClick={onClose}
           />
-          <Box position="absolute" left="calc(50% - 16.5rem)" top="0">
-            <NextImage src={MetaBox} alt="MetaBox" width={528} height={695} />
-          </Box>
+
+          <Box
+            position="absolute"
+            left="calc(50% - 16.5rem)"
+            top="0"
+            width={522}
+            height={680}
```

Comment:
Can we use responsive sizing instead of hard coded width/heights? Hardcoded sizes is why the drawer is breaking / looking weird on a lot of devices

Reply:
Things are now in terms of `rem`s. String interpolation is used along with `calc()` to create comprehensible sizing instructions.

It's feasible that some more of those statements could be derived from `flex` or `grid` layouts, but the current implementation works.